modules/spotxBidAdapter.js

```diff
@@ -83,7 +83,6 @@ export const spec = {
 
       const ext = {
         sdk_name: 'Prebid 1+',
-        player_vendor: 'SpotXJW',
```

Comment:
We added this a few weeks ago for SpotX to track that the ad is coming from our VPB. Are you sure we want to remove this? 
CC: @mamaddox

Reply:
Same question. Here is a ticket I created about it: https://jwplayer.atlassian.net/browse/JW8-11226
Now I don't know too much about this parameter, but I know it was added here after the Prebid migration as a workaround for now.

Reply:
@mamaddox  Was this merged upstream ? I saw it was failing SpotX's unit tests. I'll take a look again.

Reply:
@karimMourra It's not merged upstream because it doesn't and has never existed there. Probably why the tests are failing. 
From what I could gather, it's a parameter that we have always implemented on our side. Before migrating to open source Prebid, we used `player_vendor` to track the SpotX ads through our own VPB solution. Once we migrated to Prebid, the param was only added back to our fork as a workaround I'm assuming. I didn't see any other Prebid adapters using a param like this and SpotX never used it in their adapter, so I'm assuming that's why the change was never merged upstream. I don't know much about it though tbh.

Reply:
Ok now I see. So @mamaddox  is VPB using our fork of Prebid.js ? 
Adding `player_vendor: 'SpotXJW'` to spotX's adapter is definitely wrong so i doubt it would be accepted upstream. We probably shouldn't be modifying a vendor's adapter.  
If our current build of VPB needs `player_vendor: 'SpotXJW'` then i'm ok with not merging my branch to our fork, and opening a PR directly with Prebid.js upstream once this PR is approved.

Reply:
@karimMourra We do use our fork of Prebid. We are currently pointing to the `spotx-min-max-duration` branch.
https://github.com/jwplayer/jwplayer-ads-header-bidding/blob/92530d12ca9b0046be466cf1489028421e1a33b9/package.json#L63-L65
You could merge this into our fork since we are using that branch, but I think we would still be in the same situation we've been in regarding this `player_vendor` param. I tried to spell out everything I knew and could find about the param in that JIRA ticket.
I do agree though that we probably shouldn't just be modifying the adapter on our end.